dotcom-rendering/src/web/components/ShowMore.importable.tsx

```diff
@@ -156,8 +155,10 @@ export const ShowMore = ({
 						margin-right: 10px;
 						color: ${neutral[100]};
 						background-color: ${neutral[7]};
+						border-color: ${neutral[7]};
 						&:hover {
-							color: ${neutral[7]};
+							background-color: ${neutral[46]};
+							border-color: ${neutral[46]};
```

Comment:
Assuming this has enough contrast with `neutral[100]`.

Reply:
yep!